source/reference/config/triggers.txt

```diff
@@ -61,6 +76,11 @@ trigger type. The following fields exist in *all* trigger configuration files:
        - ``"AUTHENTICATION"``
        - ``"SCHEDULED"``
 
+   * - | ``disabled``
+       | ``boolean``
+     - Defaults to ``false`` If ``true``, the trigger will not listen for any events and will
+       not fire.
```

Comment:
I agree the disabled naming is confusing; I think it used to be enabled, but they changed it. I would prefer to not use enabled to avoid more confusion.